src/schoology-extractor/schoology_extractor/schoology_extract_facade.py

```diff
@@ -250,22 +251,34 @@ def get_user_activities(self, section_id: int) -> pd.DataFrame:
 
         Returns
         -------
-        DataFrame containing the user activities
+        DataFrame containing the section activities
         """
 
         discussions = self._client.get_discussions(section_id)
         replies: list = []
+        mapped_replies = pd.DataFrame()
         for discussion in discussions:
             discussion_id = discussion["id"]
             replies = replies + self._client.get_discussion_replies(section_id, discussion_id)
+            sync_replies = sync.sync_resource(
+                RESOURCE_NAMES.DISCUSSION_REPLIES,
+                self._db_engine,
+                replies
+            )
+            current_replies = discussionRepliesMap.map_to_udm(pd.DataFrame(sync_replies), section_id, discussion_id)
+            if mapped_replies.empty:
+                mapped_replies = current_replies
+            else:
+                mapped_replies.append(current_replies)

```

Comment:
I think you can simplify a little with `pd.concat`:

```suggestion
            mapped_replies = pd.concat([mapped_replies, current_replies])
```